test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -0,0 +1,55 @@
+var d3 = require('d3');
+
+var Plotly = require('@src/index');
+
+/*
+ * The following test cases fail on the CircleCI
+ * most likely due to a WebGL/driver issue
```

Comment:
Is this the same issue with the bunny-hill test? If it is can we link all these to an issue?